src/handler.cpp

```diff
@@ -796,12 +792,21 @@ void affect_total(struct char_data * ch)
     }
   }
 
+  // We want the higher of either cyber+bio or magic/adept
+  sbyte aug_rea = GET_REA(ch);
+  sbyte aug_init_dice = GET_INIT_DICE(ch);
+  GET_REA(ch) = 0;
```

Comment:
The quickness and intelligence contributions to reaction get added in at the very end of all the attribute stuff, so that drugs, capping, and nerve strike modifiers all apply.

There's a question as to how GET_REAL_REA should be calculated though... SR3 rules consider cerebral boosters, muscle toner, and adept improved ability to be natural in most cases.... but not always, since toners (and probably also adept improved qui) don't apply to reaction when rigging.

Reply:
I don't have a strong opinion on that question myself, so if you're confident in this code as it stands I'm happy to merge it for testing.

Reply:
Maybe leave GET_REAL_REA alone for informational purposes, and then if we're decking or rigging, let's update GET_REA (and GET_INIT_DICE) to the appropriate values.